pkgs/top-level/ocaml-packages.nix

```
@@ -2245,7 +2245,16 @@ let
 
         google-drive-ocamlfuse = callPackage ../applications/networking/google-drive-ocamlfuse { };
 
-        hol_light = callPackage ../applications/science/logic/hol_light { };
+        hol_light = callPackage ../applications/science/logic/hol_light {
+          camlp5 =
+            if lib.versionAtLeast camlp5.version "8.04.00" then
+              camlp5.overrideAttrs (_: {
```

no need to add `(_: { ... })` as mentioned here https://github.com/NixOS/nixpkgs/blob/88a0763bd35e97e3cef2bb4ce03860880b00a7aa/doc/using/overrides.chapter.md?plain=1#L79L83

```suggestion
              camlp5.overrideAttrs {
```

---

Thanks for the hint. Done.